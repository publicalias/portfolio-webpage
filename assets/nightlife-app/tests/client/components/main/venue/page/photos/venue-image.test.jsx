"use strict";

//local imports

const VenueImage = require("../../../../../../../scripts/client/components/main/venue/page/photos/venue-image");

const { testWrapper } = require("../../../../../test-helpers");

//global imports

const { initTestSnapshot, reactTests } = require("redux/tests/react-tests");

//utilities

const { testShallow } = testWrapper(VenueImage);

const testSnapshot = initTestSnapshot(testShallow);

//setup

beforeAll(reactTests.setup);
beforeEach(reactTests.inject(VenueImage));

//venue image

test("venue image should match snapshot", () => testSnapshot());
