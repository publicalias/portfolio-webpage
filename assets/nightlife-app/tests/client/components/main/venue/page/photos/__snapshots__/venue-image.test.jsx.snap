// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`venue image should match snapshot 1`] = `""`;
